### PR TITLE
New version: KRTirtho.Flemozi version 0.2.0

### PR DESCRIPTION
**Pull request by OfficialEsco**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/145586)

### CODE REVIEW

**Comment by OfficialEsco** [comment]
```
Enter a pull request number: 145586

Manifest has the following dependencies that were not validated; ensure that they are valid:
  - Packages
      Microsoft.VCRedist.2015+.x64
Manifest validation succeeded.
Found Flemozi [KRTirtho.Flemozi] Version 0.2.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
This package requires the following dependencies:
  - Packages
      Microsoft.VCRedist.2015+.x64
(1/1) Found Microsoft Visual C++ 2015-2022 Redistributable (x64) [Microsoft.VCRedist.2015+.x64] Version 14.40.33617.1
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.visualstudio.microsoft.com/download/pr/09ee6168-4ebc-4886-a058-a668f573a552/CAA8369F99545E84E0EA83CBE0C202408F3B3A038B2E1786BB048A837A9D1522/VC_redist.x64.exe
  ██████████████████████████████  24.2 MB / 24.2 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Downloading https://github.com/KRTirtho/flemozi/releases/download/v0.2.0/Flemozi-windows-x86_64-setup.exe
  ██████████████████████████████  13.0 MB / 13.0 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : Flemozi version 0.2.0+3
Publisher     :
Version       : 0.2.0+3
ProductCode   : FBCAB0B1-8171-40FA-B28E-234B17743354_is1
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\FBCAB0B1-8171-40FA-B28E-234B17743354_is1
Uninstall     : winget uninstall --id "FBCAB0B1-8171-40FA-B28E-234B17743354_is1"

Name          : Microsoft Visual C++ 2015-2022 Redistributable (x64) - 14.40.33617
Publisher     : Microsoft Corporation
Version       : 14.40.33617.1
ProductCode   : {551c9ee3-34e4-4f63-891b-7616e726a705}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\{551c9ee3-34e4-4f63-891b-7616e726a705}
Uninstall     : winget uninstall --id "{551c9ee3-34e4-4f63-891b-7616e726a705}"

Press any key to continue . . .
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> 2024-03-20 20:12:51.283 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFC2A4CC11F: (caller: 00007FFC2A258D6F) LogHr(1) tid(874) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFC2A3D2B08: (caller: 00007FFC2A47BC20) Exception(1) tid(874) 8A15005E ] 
> 2024-03-20 20:12:51.283 [REPO] Failed to open available source: msstore
> 2024-03-20 20:12:54.345 [CLI ] Caught wil::ResultException: D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFC2A3D2B08: (caller: 00007FFC2A47BC20) Exception(1) tid(874) 8A15005E 
> 2024-03-20 20:13:02.872 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\RepositorySource.cpp(757)\WindowsPackageManager.dll!00007FFC0CD7C11F: (caller: 00007FFC0CB08D6F) LogHr(1) tid(b84) 8A15005E     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerRepositoryCore\Rest\Schema\HttpClientHelper.cpp(30)\WindowsPackageManager.dll!00007FFC0CC82B08: (caller: 00007FFC0CD2BC20) Exception(1) tid(b84) 8A15005E ] 
> 2024-03-20 20:13:02.872 [REPO] Failed to open available source: msstore
> 2024-03-20 20:13:04.917 [CLI ] No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.40.33617.1

(Automated response - build 870.)

**Comment by OfficialEsco** [comment]
> No suitable installer found for manifest Microsoft.VCRedist.2015+.x64 with version 14.40.33617.1

Is it trying to run on x86 when the installer says x64?

File: manifests/k/KRTirtho/Flemozi/0.2.0/KRTirtho.Flemozi.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: KRTirtho.Flemozi
+PackageVersion: 0.2.0
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
Comment (stephengillie) [inline manifests/k/KRTirtho/Flemozi/0.2.0/KRTirtho.Flemozi.installer.yaml:11]:
```suggestion
```

File: manifests/k/KRTirtho/Flemozi/0.2.0/KRTirtho.Flemozi.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/KRTirtho/flemozi/releases/download/v0.2.0/Flemozi-windows-x86_64-setup.exe
+  InstallerSha256: 2A1DB28077D01E014E7573C6AE37D6CAD482DB4823C13C35FED2A5406B90011F
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/KRTirtho/flemozi/releases/download/v0.2.0/Flemozi-windows-x86_64-setup.exe
+  InstallerSha256: 2A1DB28077D01E014E7573C6AE37D6CAD482DB4823C13C35FED2A5406B90011F
```
Comment (stephengillie) [inline manifests/k/KRTirtho/Flemozi/0.2.0/KRTirtho.Flemozi.installer.yaml:27]:
```suggestion
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
  InstallerSha256: 2A1DB28077D01E014E7573C6AE37D6CAD482DB4823C13C35FED2A5406B90011F
```

File: manifests/k/KRTirtho/Flemozi/0.2.0/KRTirtho.Flemozi.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ReleaseDate: 2023-09-09
+AppsAndFeaturesEntries:
+  - ProductCode: "FBCAB0B1-8171-40FA-B28E-234B17743354_is1"
+    DisplayVersion: "0.2.0+3"
+    DisplayName: "Flemozi version 0.2.0+3"
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/KRTirtho/flemozi/releases/download/v0.2.0/Flemozi-windows-x86_64-setup.exe
+  InstallerSha256: 2A1DB28077D01E014E7573C6AE37D6CAD482DB4823C13C35FED2A5406B90011F
```
Comment (stephengillie) [inline manifests/k/KRTirtho/Flemozi/0.2.0/KRTirtho.Flemozi.installer.yaml:21]:
```suggestion
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
  InstallerSha256: 2A1DB28077D01E014E7573C6AE37D6CAD482DB4823C13C35FED2A5406B90011F
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> Is it trying to run on x86 when the installer says x64?

Edit2: Looking at this again, I'm not sure what it's doing.

**Comment by stephengillie** [comment]
Hi @OfficialEsco,

This manifest adds Apps and Features entries that aren't present in previous PR versions. These entries should be added to the previous versions, or removed from this version.

(Automated response - build 870.)

**Comment by OfficialEsco** [comment]
Here u go #145586

**Pull request closed without merging**